test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -926,5 +929,11 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
             Assert.NotNull(transferModeFeature);
             Assert.Equal(TransferMode.Binary, transferModeFeature.TransferMode);
         }
+
+        private bool IsNegotiateRequest(HttpRequestMessage request)
+        {
+            return request.Method == HttpMethod.Post &&
+                request.RequestUri.ToString().Contains("/negotiate");
```

Comment:
Shouldn't be something like `Path.EndsWith("/negotiate")` instead of `Contains`?